### PR TITLE
Update deps

### PR DESCRIPTION
**Pull request by charleskawczynski**
This PR updates the dependencies

### CODE REVIEW

File: docs/Manifest.toml [hunk 1]
```diff
@@ -164,9 +164,9 @@ version = "0.8.0"
 
 [[deps.ClimaComms]]
 deps = ["CUDA", "StaticArrays"]
-git-tree-sha1 = "93c4c545024821c9cacb7417f00381b2329e76db"
+git-tree-sha1 = "5ab2642608e411eaf96d5d59c49ae4ab3bcdf2df"
 uuid = "3a4d1b5c-c61d-41fd-a00a-5873ba7a1b0d"
-version = "0.3.2"
+version = "0.3.4"
 
 [[deps.ClimaCore]]
```
Comment (valeriabarra) [inline docs/Manifest.toml:171]: Should we wait for the new ClimaCore release?

Comment (charleskawczynski) [reply docs/Manifest.toml:171]: We may have to wait. The PR has several failures that need to be resolved.

File: docs/Manifest.toml [hunk 2]
```diff
@@ -164,15 +164,15 @@ version = "0.8.0"
...
 deps = ["Adapt", "BlockArrays", "CUDA", "ClimaComms", "CubedSphere", "DataStructures", "DiffEqBase", "DocStringExtensions", "ForwardDiff", "GaussQuadrature", "GilbertCurves", "HDF5", "InteractiveUtils", "IntervalSets", "LinearAlgebra", "PkgVersion", "RecursiveArrayTools", "RootSolvers", "Rotations", "SparseArrays", "Static", "StaticArrays", "Statistics", "UnPack"]
-git-tree-sha1 = "ba403ef47480ab31edb6b02d499538574b321c74"
+git-tree-sha1 = "db9de456563882f1febe1436cc4f9785d3d21be7"
 uuid = "d414da3d-4745-48bb-8d80-42e94e092884"
-version = "0.10.23"
+version = "0.10.24"
```
Comment (valeriabarra) [inline docs/Manifest.toml:175]: :tada:

**Comment by szy21** [comment]
@charleskawczynski I just released SurfaceFluxes 0.3.5, could you update it as well? I tested this version and it shouldn't require any change in the MSE table: https://buildkite.com/clima/climaatmos-ci/builds/6411

**Comment by charleskawczynski** [comment]
Tracking progress over at https://github.com/SciML/DiffEqBase.jl/issues/863

**Comment by simonbyrne** [comment]
Blocked by RRTMGP.jl changes:
https://github.com/CliMA/ClimaAtmos.jl/blob/b4cd235c30db7e481c6b3b22e5f888abac969025/src/RRTMGPInterface.jl#L1231
is now defined in RRTMGP.jl:
https://github.com/CliMA/RRTMGP.jl/blob/66cf373933eddd1fdd0c1c0f70db7a3a654ff397/src/optics/GrayOpticsKernels.jl#L17

@szy21 let me know what we should do

**Comment by valeriabarra** [comment]
bors try

**Comment by simonbyrne** [comment]
https://buildkite.com/clima/climaatmos-ci/builds/6495#0185c5e0-f2a2-48e8-b275-31155f998074/137-206
```
maxiter reached in saturation_adjustment:
    Method=NewtonsMethod, T_guess=nothing, e_int=NaN, 
ρ=1.1645852428602015, q_tot=0.016980813575153148, T=NaN, maxiter=8, 
tol=0.0001
```

**Comment by simonbyrne** [comment]
I added a fix to DiffEqBase in https://github.com/SciML/DiffEqBase.jl/pull/867

**Comment by szy21** [comment]
Thanks @simonbyrne! Should we merge this now?

**Pull request merged**